File: Sort_Python/Selection_Sort.py
def Selection(A):
    n = len(A)
    for i in range(0, n-1):
        i_min = i
        v_min = A[i]
        for j in range(i, n):
            if (A[j] < v_min):
                v_min = A[j]
                i_min = j
        A[i], A[i_min] = A[i_min], A[i]
n = int(input("Nhập số lượng phần tử của mảng: "))
A = []
for i in range(n): 
    A.append(int(input(f"Nhập phần tử thứ {i} của mảng: ")))
Selection(A)
print(A)

File: ConsoleApp1/Sort_New/Selection_Sort.cs

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Sort_New
{
    internal class Selection_Sort
    {
        static void Selection(int[] A, int n)
        {
            for (int i = 0; i < n; i++)
            {
                int i_min = i;
                int V_min = A[i];
                for (int j = i; j < n; j++)
                {
                    if (A[j] < V_min)
                    {
                        V_min = A[j];
                        i_min = j;
                    }
                }
                int temp = A[i];
                A[i] = A[i_min];
                A[i_min] = temp;
            }
        }
        static void Main(string[] args)
        {
            Console.Write("Input n: ");
            int n = Convert.ToInt32(Console.ReadLine());
            int[] A = new int[n];
            Console.WriteLine("ADD item of array:");
            for (int i = 0; i < n; i++)
            {
                Console.Write($"Array[{i}]");
                A[i] = Convert.ToInt32(Console.ReadLine());
            }
            Selection(A, n);
            Console.WriteLine("Array after sort:");
            foreach (int i in A)
            {
                Console.Write(i + " ");
            }
        }
    }
}
